.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 name: "MiNiFi-CPP CI"
 on: [push, pull_request, workflow_dispatch]
 jobs:
-  macos_xcode_10_3:
-    name: "macos-xcode10.3"
+  macos_xcode_11_2_1:
+    name: "macos-xcode11.2.1"
```

Comment:
I'm not sure there is too much value in running two builds using the same major version. 

I would recommend one to utilize xcode 12:
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#xcode

Reply:
I have replaced 11.7 with 12.0